lib/jekyll-seo-tag/url_helper.rb

```diff
@@ -8,8 +8,8 @@ module UrlHelper
 
       # Determines if the given string is an absolute URL
       #
-      # Returns true if an absolute URL.
```

Comment:
Why was this *period* removed, @MichaelCurrin?

Reply:
The two lines below and the one above do not have one. So made it consistent.

Reply:
Fair enough.